.github/workflows/sims-045.yml

```diff
@@ -7,17 +7,9 @@ on:
   release:
     types: [published]
 
+concurrency: ci-${{ github.ref }}
```

Comment:
this based off commit or?

Reply:
No, this is based on branch name, prs and tags: like `refs/heads/<branch_name>`, `refs/pull/<pr_number>/merge` `refs/tags/<tag_name>`.